test/cases/ostree.sh

```diff
@@ -33,7 +33,7 @@ case "${ID}-${VERSION_ID}" in
         IMAGE_TYPE=rhel-edge-commit
```

Comment:
Can you also change the value of line 31 above to download.devel and remove the comment above it.